### DISCUSSION TITLE
We need more Windows testers, especially those running Windows 11!

### DISCUSSION THREAD

**Discussion by Pentaphon** [transmission/transmission#3709] [General]
I've found some obvious issues on Windows and I think we need more people testing on Windows especially now that 4.0.0 beta 1 is close to release. 

1) https://github.com/transmission/transmission/issues/3708

2)  https://github.com/transmission/transmission/issues/1265

**Comment by branhillsdesign** [comment]
Just installed with Firefox open on Windows 11. No issues at all. Running Transmission 4.0.0-beta.1

**Reply by Pentaphon** [reply]
> Just installed with Firefox open on Windows 11. No issues at all. Running Transmission 4.0.0-beta.1

I hope somebody actually fixed the installer then because when I bring this up, nobody seems to confirm they issued a fix.

**Comment by spupuz** [comment]
what about docker version?

**Reply by Pentaphon** [reply]
> what about docker version?

Probably more important to fix the Windows version. Even the installer on 4.0.0-beta2 seems to be breaking Firefox.
